CHANGELOG.md

```diff
@@ -6,9 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [unreleased]
- ### Added
+ ### Added 
  ### Changed 
  ### Fixed 
+ - Fix oauth for admin and management ([#1196](https://github.com/ehrbase/ehrbase/pull/1160](https://github.com/ehrbase/ehrbase/pull/1196))
```

Comment:
I would change to wording to make it more clear
`- Fix OAuth2 Security Configuration for admin role and management endpoints`

Also, there are 2 links in there.